### PR TITLE
Bring back missing tracing and monitoring tests 

### PR DESCRIPTION
**Pull request by VladislavPaskar**
**Description**

Changes proposed in this pull request:

- bring back the tests that got delete due refactoring in https://github.com/kyma-project/kyma/pull/12905

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/12846

the prowjob PR: https://github.com/kyma-project/test-infra/pull/4834

### CODE REVIEW

**Review by pxsalehi — DISMISSED** [review]

File: tests/fast-integration/eventing-test/eventing-test-prep.js [hunk 1]
```diff
@@ -112,9 +111,9 @@ describe('Eventing tests preparation', function() {
     );
   }
 
-  after(async function() {
+  afterEach(async function() {
     // if the test preparation failed, perform the cleanup
-    if (this.currentTest.err && this.currentTest.err.code === fatalErrCode) {
+    if (this.currentTest.state === 'failed') {
```
Comment (mfaizanse) [inline tests/fast-integration/eventing-test/eventing-test-prep.js:116]: Why this is in `afterEach`? Should it be in `after` where we get the overall test results?

**Review by pxsalehi — APPROVED** [review]

**Review by pxsalehi — APPROVED** [review]

**Pull request merged**